doc/INSTALL.md

```diff
@@ -22,74 +22,115 @@ For the following Linux distributions and package managers 0.8 packages are avai
 
-### Ubuntu-Bionic
+We also provide more recent versions and bleeding-edge versions in our repositories:
 
----
+### Debian/Ubuntu
 
-To use the Ubuntu Bionic repository of the latest builds from master following steps need to be made:
+We provide stable and bleeding-edge repositories for following Debian-based distributions:
 
-1. Run `sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys D919CE8B27A64C16656FCA9FF1532673651F9C6C` to obtain the key.
+- Debian Buster
+- Ubuntu Focal
+- Ubuntu Bionic
 
-2. Add `deb https://ubuntu-bionic-repo.libelektra.org/ bionic main` into `/etc/apt/sources.list`
+To use our stable repositories with our recent releases, following steps need to be made:
 
-   Which can also be done using:
+1. Run `sudo apt-key adv --keyserver keys.gnupg.net --recv-keys F26BBE02F3C315A19BF1F791A9A25CC1CC83E839` to obtain the key.
 
-   ```sh
-   apt-get install software-properties-common apt-transport-https
-   echo "deb https://ubuntu-bionic-repo.libelektra.org/ bionic main" | sudo tee /etc/apt/sources.list.d/elektra.list
-   ```
+2. Add `deb https://debs.libelektra.org/<DISTRIBUTION> <DISTRIBUTION> main` into `/etc/apt/sources.list`
+   where `<DISTRIBUTION>` is the Codename of your distributions e.g.`focal`,`bionic`,`buster`, etc.
 
-   Or alternatively, you can use (if you do not mind many dependencies just to add one line to a config file):
+This can also be done using:
 
-   ```sh
-   sudo apt-get install software-properties-common apt-transport-https
-   sudo add-apt-repository "deb https://ubuntu-bionic-repo.libelektra.org/ bionic main"
-   ```
-
-### Debian-Buster
+```sh
+# Example for Ubuntu Focal
+apt-get install software-properties-common apt-transport-https
+echo "deb https://debs.libelektra.org/focal focal main" | sudo tee /etc/apt/sources.list.d/elektra.list
+```
 
-To use the Debian Buster repository of the latest builds from master put following lines in
-`/etc/apt/sources.list`:
+Or alternatively, you can use (if you do not mind many dependences just to add one line to a config file):
 
+```sh
+# Example for Ubuntu Focal
+sudo apt-get install software-properties-common apt-transport-https
+sudo add-apt-repository "deb https://debs.libelektra.org/focal focal main"
 ```
-deb     [trusted=yes] https://debian-buster-repo.libelektra.org/ buster main
-deb-src [trusted=yes] https://debian-buster-repo.libelektra.org/ buster main
+
+If you would like to use our bleeding-edge builds (master builds), append `-unstable` to `<DISTRIBUTION>`.
+
+The `etc/apt/source.list` entry must look like following: `deb https://debs.libelektra.org/<DISTRIBUTION> <DISTRIBUTION>-unstable main`
+
+E.g. `deb https://debs.libelektra.org/focal focal-unstable main`
+
+> NOTE: for Ubuntu Bionic the yamlcpp plugin is excluded due to missing dependencies and therefore the package `libelektra4-yamlcpp` is not available.
```

Comment:
Is this really the only plugin (or tool or binding) which is excluded? Did you check in the log if everything else is included?

Reply:
I checked the logs and all plugins/bindings/tools that are part of a package are included on all distros except for the yamlccp plugin on Ubuntu Bionic.

Reply:
Thank you so much for checking this! We should also add a TODO task for the releases to check this for every release.